src/App.js

```diff
@@ -31,9 +41,10 @@ function App() {
   return (
     <>
       <h1>Todo List</h1>
-        <AddTodoForm onAddTodo={addTodo} />
+      <AddTodoForm onAddTodo={addTodo} />
       <hr />
-       <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
+      {isLoading ? <p>Page Is Loading....</p> : TodoList}
+      <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
```

Comment:
Your app is currently working properly and here is the reason why. It always displays the `TodoList`, does conditionally render the loading text, and has an error in the console. This is happening because on line 46 you're telling it that depending on `isLoading`, you either want to display a `<p>` or the actual function `TodoList`, which is causing that error to pop. You also have `<TodoList>` outside of the ternary so it's not being conditionally rendered. Below is the proper code, please let me know if you have any questions about this.

```suggestion
      {isLoading ? (
        <p>Page Is Loading....</p>
      ) : (
        <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
      )}

```